integration/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Custom Integration
+---
+
+# Custom Integration
+
+This section describes the integration between existing or custom implementations of observatory software components.
+
+## Telescope Control System (TCS)
+
+A TCS provides the interface between the telescope subsystems and the Observation Portal. [This document]({% link integration/tcs.md %}) describes, in detail, the necessary interactions between the OCS Observation Portal and an observatory's TCS software.
+
+Most observatories will have existing TCS software, so this document is written in order to present the capabilities that will need to be integrated into existing software so that it can communicate with the OCS Observation Portal.
+
+## Custom Scheduler
+
+[This document]({% link integration/scheduler.md %}) describes the role of the observatory scheduling software and presents a general overview of the role of a scheduler - to schedule observations on observatory resources based on existing requests from observatory users or staff.
```

Comment:
For some reason these links aren't working correctly on my live fork - they work just fine spun up locally